src/main/msp/msp_serial.h

```diff
@@ -83,11 +83,11 @@ typedef struct mspPort_s {
     struct serialPort_s *port; // null when port unused.
     mspState_e c_state;
     uint8_t inBuf[MSP_PORT_INBUF_SIZE];
-    uint16_t cmdMSP;
-    uint8_t cmdFlags;
+    int offset;
+    int dataSize;
     mspVersion_e mspVersion;
```

Comment:
Why not fixed width types instead of int for `offset` and `dataSize`? From the code, it seems that they both would always fit in an uint16_t.

Reply:
Using a generic type will allow compiler to optimize better. Using a fixed data type might force compiler to enforce that type by using narrow instructions.

Reply:
If you want the compiler to optimize for speed there, I'd use `uint_fast16_t`. I guess we're unlikely to start supporting platforms where int is 16 bits so it should be fine either way, but I'd make them at least unsigned for readability since they are both only assigned to unsigned values.

Reply:
Good point